src/testE2E/codecatalyst/client.test.ts

```diff
@@ -81,7 +81,7 @@ let projectName: CodeCatalystProject['name']
  *     integ tests, but using the ssh hostname that we get from
  *     {@link prepareDevEnvConnection}.
  */
-describe('Test how this codebase uses the CodeCatalyst API', function () {
+describe.skip('Test how this codebase uses the CodeCatalyst API', function () {
```

Comment:
Perhaps we could catch exception and skip only if the error matches `ThrottlingException`.

A data-based check is also ok, but 1 month is probably too short.